src/infrastructure/persistence/PsqlReportRepository.test.ts

```diff
@@ -0,0 +1,60 @@
+import database from '../../database';
+import { PsqlReportRepository } from './PsqlReportRepository';
+import { PsqlTestRepository } from './PsqlTestRepository';
+
+import { Test } from '../../domain/model/test/Test';
+import { TestId } from '../../domain/model/test/TestId';
+import { Results } from '../../domain/model/test/Results';
+import { UserId } from '../../domain/model/user/UserId';
+
+import { cleanupDatabase } from '../../test/cleanupDatabase';
+import { ConfidenceLevel } from '../../domain/model/test/ConfidenceLevel';
+import { aTest, aTestType } from '../../test/domainFactories';
+import { testTypeRepository } from './';
+
+describe('PsqlReportRepository', () => {
+  const psqlReportRepository = new PsqlReportRepository(database, testTypeRepository);
+  const psqlTestRepository = new PsqlTestRepository(database, testTypeRepository);
+
+  beforeEach(async () => {
+    await cleanupDatabase();
+  });
+
+  it('inserts a test without results and gets report', async () => {
```

Comment:
I guess here we aren't testing the insert. Maybe this should read: "gets results for a single test"?